tools/roslaunch/src/roslaunch/launch.py

```diff
@@ -386,7 +388,9 @@ def _launch_master(self):
             validate_master_launch(m, self.is_core, self.is_rostest)
 
             printlog("auto-starting new master")
-            p = create_master_process(self.run_id, m.type, get_ros_root(), m.get_port())
+            num_workers = getattr(self, "num_workers", NUM_WORKERS)
+            timeout = getattr(self, "timeout", None)
```

Comment:
With these attributes being members of the class `getattr` is not necessary here.